less/mixins.less

```diff
@@ -1 +1,8 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
```

Comment:
I know they may seem like overkill on such a small project, but would have loved to see you attempt another mixin or two!